Highcharts | Improve examples & docs (#5875)
jpmorganchase/salt-ds: site/docs/components/chart/examples.mdx

Reviewer: ```suggestion
Use a bubble chart to show the relationship between two variables, with a third variable represented by the size of each bubble. This is helpful for visualizing clusters and outliers across categories.
```

---

<Callout title="Note">
  <ul>
    <li>
      To ensure the presentation is accessible, line or fill patterns can be
      applied to the chart (see [Patterns and Fills](./usage#patterns-and-fills)
      for details). Use the "Show patterns" switch to toggle patterns.
    </li>
    <li>
      This package is a WIP. Please check the [current
      issues](https://github.com/jpmorganchase/salt-ds/issues/5878) for known
      issues/gaps.
    </li>
  </ul>
</Callout>

## Line chart

Use a line chart to show changes in data over time.

<LivePreview componentName="chart" exampleName="LineChart" />

## Pie chart

Use a pie chart to show proportions of a whole.

<LivePreview componentName="chart" exampleName="PieChart" />

## Donut chart

Use a donut chart to show proportions of a whole, with space in the center to highlight key values or additional information.

A donut chart is a variation of the pie chart, with the `plotOptions.pie.innerSize` set to less than `100%` within the `Options` object.

### Best practices

> - An `innerSize` value of `80%` is recommended for a good default visual balance.

Reviewer: ```suggestion
An `innerSize` value of `80%` is recommended for a good default visual balance.
```
Nitpick: Probably not necessary for single item

---

<Callout title="Note">
  <ul>
    <li>
      To ensure the presentation is accessible, line or fill patterns can be
      applied to the chart (see [Patterns and Fills](./usage#patterns-and-fills)
      for details). Use the "Show patterns" switch to toggle patterns.
    </li>
    <li>
      This package is a WIP. Current issues are [tracked

>       here](https://github.com/jpmorganchase/salt-ds/issues/5878).

Reviewer 2: ```suggestion
      This package is a WIP. Please check the [current issues](https://github.com/jpmorganchase/salt-ds/issues/5878) for known issues/gaps.
```

  below.
</Callout>

<LivePreview componentName="chart" exampleName="BoxPlotChart" />

## Area chart

Use an area chart to show trends over time and the contribution of categories to a total.

<LivePreview componentName="chart" exampleName="AreaChart" />

## Scatter chart

> Use a scatter chart to show the relationship between two numerical variables. To ensure accessible presentation, fill patterns can be applied to the scatter points (see [Patterns and Fills](./usage#patterns-and-fills) for details).

Reviewer: ```suggestion
Use a scatter chart to show the relationship between two numerical variables.
```
